src/Components/Members/MemberEditCreate.js

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { CKEditor } from "@ckeditor/ckeditor5-react";
+import ClassicEditor from "@ckeditor/ckeditor5-build-classic";
+import Swal from "sweetalert2";
+import "../FormStyles.css";
+
+const MemberEditCreate = () => {
+  const [member, setMember] = useState({
+    name: "",
+    description: "",
+    imageMember: "",
+    social_media: "",
+  });
+
+  const handleChange = (e) => {
+    if (e.target.name === "name") {
+      setMember({ ...member, name: e.target.value });
+    }
+
+    if (e.target.name === "image") {
+      const fileValid =
```

Comment:
Maybe this can const can be a function on Utils/validation. The function can be named isValidImage

Reply:
changes added